src/components/render.js

```diff
@@ -313,6 +313,7 @@ class CanvasRender extends Component {
 
     return (
       <React.Fragment>
+        { renderError && <RenderError message={"Invalid image!"}/> }
```

Comment:
If the RenderError is visible we should hide the openseadragon `viewer`.

When a `renderError` occurs we should disable any interaction with the viewer otherwise it will break the component.

Reply:
You want to stop the viewer on renderError? I mean, if a image is bad you want to prevent the user to see the other images or only the bad one